### DISCUSSION TITLE
Header switch rocks 🚀

### DISCUSSION THREAD

**Discussion by noraj** [glitchedgitz/grroxy-issues#21] [Show and tell]
This feature of enabling / disabling a line / header is original and so nice.

![image](https://github.com/glitchedgitz/grroxy-issues/assets/16578570/16751b12-84b5-4d2f-9537-c8ac593c16e2)

**Comment by glitchedgitz** [comment]
Thanks! You can also use shortcut `Ctrl` `/`😉

**Reply by noraj** [reply]
Also it would be nice when there will be shortcut remaping in settings because on AZERTY keyboard `/` is a secondary option on the key so I actually need to hiy `CTRLL` + `SHIFT` + `/`

**Reply by glitchedgitz** [reply]
Sure, but it will be for later stages.
